manifests/cf-manifest/operations.d/540-log-cache.yml

```diff
@@ -0,0 +1,228 @@
+# FIXME: Copied from cf-deployment experimental opsfile in the latest
+# version of cf-deployment
+#
+# https://github.com/cloudfoundry/cf-deployment/blob/v1.31.0/operations/experimental/use-log-cache.yml
+#
+# Replace for the upstream opsfile once we upgrade cf-deployment > v1.31.0
+#
+- type: replace
+  path: /releases/-
+  value:
+    name: log-cache
+    sha: 87134510c23ce5facca665ac025f41a19340f5e9
+    url: https://bosh.io/d/github.com/cloudfoundry/log-cache-release?v=1.1.1
+    version: 1.1.1
+
+- type: replace
+  path: /instance_groups/name=doppler/jobs/-
+  value:
+    name: log-cache
+    properties:
+      egress_port: 8080
+      health_addr: localhost:6060
+      tls:
+        ca_cert: ((log_cache.ca))
```

Comment:
This CA cert (and 2 other below) are missing the `_old` variant, meaning that CA cert rotation currently breaks log-cache. These were missed because the existing test for CA certs doesn't account for access via the `cert.ca` variables.

I'll add a fix for this (and update the test) in #1379.